docs/entity-manager-api.md

```diff
@@ -206,14 +213,23 @@ await manager.upsert(
  **/
 ```
 
--   `delete` - Deletes entities by entity id, ids or given conditions:
+-   `delete` - Deletes entities by entity id, ids or given conditions.
 
 ```typescript
 await manager.delete(User, 1)
 await manager.delete(User, [1, 2, 3])
 await manager.delete(User, { firstName: "Timber" })
 ```
 
+-   `deleteAll` - Deletes *all* entities of target type (without WHERE clause).
```

Comment:
I wonder if this needs more of a warning since it can be so destructive? The naming is quite clear though IMO.

Reply:
I guess we could take [a leaf from React](https://legacy.reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) and call it `dangerouslyDeleteAll`.

That might be excessive coddling from an ORM though 😀

Reply:
Ha that would definitely get the point across but it does seem excessive. I meant more of a warning via the docs, but what you wrote is clear so this seems fine.